routes/web.php

```diff
@@ -39,6 +39,8 @@
     Route::get('/chirps/recent', [ChirpController::class, 'recent'])->name('recent');
     Route::get('/chirpers',[ProfileController::class, 'index'])->name('chirpers.index');
     Route::get('/profile/{username}', [ProfileController::class, 'show'])->name('profile.show');
+    Route::post('profile/{user_id}', [ProfileController::class, 'follow'])->name('profile.follow');
```

Comment:
these routes are duplicate. the second route will never be touched because it will always match the first route. a URI and method (post, get, etc...) need to be unique. change to 'profile/{user_id}/follow'  'profile/{user_id}/unfollow'